x/stake/keeper/key.go

```diff
@@ -34,6 +34,17 @@ var (
 
 const maxDigitsForAccount = 12 // ~220,000,000 atoms created at launch
 
+// Formats a time.Time into a []byte that can be sorted
+func FormatTimeKey(t time.Time) []byte {
+	return []byte(t.UTC().Round(0).Format("2006-01-02T15:04:05.000000000"))
+}
+
+// Parses a []byte encoded using FormatTimeKey back into a time.Time
+func ParseTimeKey(bz []byte) (time.Time, error) {
```

Comment:
{Format,Parse}TimeKey are the same {Format,Parse}TimeBytes, ain't they?

Reply:
Fixed in #2517